packages/ann/ann/c_src/logistic_actf_component.h

```diff
@@ -46,6 +46,26 @@ namespace ANN {
 
     virtual char *toLuaString();
   };
+
+  class SparseLogisticActfANNComponent: public LogisticActfANNComponent {
```

Comment:
Add here doxygen documentation with a reference to Andrew Ng document with the problem statement